### ISSUE TITLE
3d-tiles branch: trackedEntity doesn't work?

### ISSUE THREAD

**Issue by kainino0x** [AnalyticalGraphicsInc/cesium#2936]
I've only compared `3d-tiles` with `mesh_compression_open3dgc`, but when I use `trackedEntity` in `3d-tiles` it just centers the camera at the center of the Earth, as if without `trackedEntity`.

I can provide more details later after SIGGRAPH if needed.

```
function createModel(url) {
    viewer.entities.removeAll();

    var position = Cesium.Cartesian3.fromDegrees(0, 0, 0);
    var orientation = Cesium.Transforms.headingPitchRollQuaternion(position, 0, 0, 0);

    var entity = viewer.entities.add({
        name : url,
        position : position,
        orientation : orientation,
        model : {
            uri : url,
            minimumPixelSize : 128
        }
    });
    viewer.trackedEntity = entity;
}
```

**Comment by pjcozzi** [comment]
Most likely, this is the problem: https://github.com/AnalyticalGraphicsInc/cesium/compare/3d-tiles#diff-b63bb2797bd8e2bc3fc273f1d6b1bca0R120

We don't plan to keep the display condition change so feel free to delete all the changes in that file.

**Comment by kainino0x** [comment]
Changing that to `false` fixes it. I really only need this right now for debugging and demo.

**Comment by pjcozzi** [comment]
Fixed in 741c1adbceb193ae7d78b1e998a8012020c392d0.
